README.md

```diff
@@ -277,7 +276,7 @@ Note: This tutorial requires that a SQL database is setup as shown in [Create a
 - Open your app in VSCode
 - Press 'F1' and search for 'Azure Functions: Create Function'
 - Choose HttpTrigger ->  (Provide a function name) -> Company.namespace is fine -> anonymous
-- In the file that opens, replace the 'public static async Task<IActionResult> Run' block with the below code
+- In the file that opens, replace the 'public static async Task&lt;IActionResult&gt; Run' block with the below code
```

Comment:
I'd suggest using backticks here anyways so that the code content is clearly separated from the rest of the sentence.